handbook/company/leadership.md

```diff
@@ -32,24 +32,17 @@ If you have any questions or feedback, please contact us: https://fleetdm.com/ha
 
 - `# Name of department`
   - "This handbook page details processes specific to working `[with](#contact-us)` and `[within](#responsibilities)` this department." 
-    
-  - `## What we do`
-    - Outline the specific responsibilities of the department.  
 
   - `## Team`
     - Table that displays each position and the team member(s) that fill that position, linking each Fleetie's LinkedIn to their name and GitHub to GitHub user name. See [handbook/ceo#team](https://fleetdm.com/handbook/ceo#team) for example.
 
-
   - `## Contact us`
-    - "To make a request of this department, `[create an issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23{DEPARTMENTAL-GITHUB-LABEL}&projects=&template=custom-request.md&title=Request%3A+_______________________)` and a team member will get back to you within one business day."
-      - "Please use issue comments and GitHub mentions to communicate follow-ups or answer questions related to your request."
+    - "To make a request of this department, `[create an issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23{DEPARTMENTAL-GITHUB-LABEL}&projects=&template=custom-request.md&title=Request%3A+_______________________)` and a team member will get back to you within one business day (If urgent, mention a `[team member](#team)` in `[#g-DEPARTMENTAL-SLACK-CHANNEL]({DEPARTMENTAL-SLACK-CHANNEL-LINK})`)."
+      - "Please **use issue comments and GitHub mentions** to communicate follow-ups or answer questions related to your request."
```

Comment:
This is called from the outside in, so it's ok for it to be generic and short.

Unlike a function in code, the department doesn't specify every kind of request they can take from the outside. Instead, they can handle all sorts of requests.

There are times when they'll have specific ways of sending stuff in, but the general philosophy (see "why this way" about bug issue template) is that the place where the need for contacting this department for a given situation arises (elsewhere in the handbook) is the place where the "To do XY, create a issue in #g-d.  Be sure to include your … yada yada"  (And expectations about response time live there)


If a department wants to better restrict intake through the use of a validated form, they can do that.  (Currently issue templates are used in this way)